cli/setup-arcos.sh

```diff
@@ -28,3 +28,4 @@ rack nodegroups import ../nodegroups/ingestion/arcos.arbiter
 # rack data import --clear ../RACK-Ontology/OwlModels/ARP-4754A.yaml     # from datagraph http://rack001/arp-4754a
 # rack data import --clear ../RACK-Ontology/OwlModels/DO-330.yaml        # from datagraph http://rack001/do-330
 # rack data import --clear ../RACK-Ontology/OwlModels/DO-178C.yaml         # from datagraph http://rack001/do-178c
+rack data import --clear ../RACK-Ontology/OwlModels/MITRE-CWE.yaml     # from datagraph http://rack001/mitre-cwe
```

Comment:
Eric, this confuses me. Why is it 'rack data import' but pointing to the OwlModels; shouldn't it be 'rack model import'?
Of course there also needs to be a 'rack data import' but that would be using the nodegroups instead, right?

Reply:
This is confusing because it's wrong. I'll fix it